doc/release-notes.md

```diff
@@ -20,361 +22,179 @@ How to Upgrade
 If you are running an older version, shut it down. Wait until it has completely
 shut down (which might take a few minutes for older versions), then run the
 installer (on Windows) or just copy over /Applications/Dash-Qt (on Mac) or
-dashd/dash-qt (on Linux).
+dashd/dash-qt (on Linux). Because of per-UTXO fix (see below) there is a
+one-time database upgrade operation, so expect a slightly longer first run.
 
 Downgrade warning
 -----------------
-### Downgrade to a version < 0.12.2
 
-Because release 0.12.2 includes DIP0001 (2 MB block size hardfork) plus
-a transaction fee reduction and a fix for masternode rank calculation algo
-(which activation also depends on DIP0001) this release will not be
-backwards compatible after DIP0001 lock in/activation happens.
+### Downgrade to a version < 0.12.2.2
+
+Because release 0.12.2.2 includes per-UTXO fix (see below) which changes the
+structure of the internal database, this release is not backwards compatible.
+You will have to reindex database if you decide to use previous version.
```

Comment:
Because release 0.12.2.2 includes **the** per-UTXO fix (see below)**,** which changes the
structure of the internal database, this release is not **fully** backwards compatible.
You will have to reindex **the** database if you decide to use **the** previous version.